shared/java/com/google/idea/blaze/common/Label.java

```diff
@@ -57,17 +56,22 @@ public static Label of(String label) {
   }
 
   public static Label fromWorkspacePackageAndName(String workspace, Path packagePath, Path name) {
```

Comment:
add test too, to see that it works with `WindowsPath`?

Reply:
You mean the subclass of `java.nio.file.Path`? Unfortunately, it only exists on Windows JVMs. However, I have verified manually that my changes fix the issue on Windows.